doc/start/quick-ceph-deploy.rst

```diff
@@ -40,13 +40,10 @@ Create a Cluster
 If at any point you run into trouble and you want to start over, execute
 the following to purge the configuration::
 
+	ceph-deploy purge {ceph-node} [{ceph-node}]
```

Comment:
@hikycool maybe you could shoot this question over IRC or mailing list with more details. as it's not relevant here. and i have no idea the error you have while configuring/building ceph.